packages/appsync-emulator-serverless/loadServerlessConfig.js

```diff
@@ -4,6 +4,7 @@
  */
 
 const Serverless = require('serverless');
+const AwsProvider = require('serverless/lib/plugins/aws/provider/awsProvider.js');
```

Comment:
Is this exposed anywhere else ? I am a little concerned tying this to a specific file.